all/entrypoint.sh

```diff
@@ -159,6 +169,15 @@ plugins=grpc+embedded\
         ;;
 esac
 
+LINT_STRING=''
+if [[ $LINT == true ]]; then
+    if [[ $LINT_CHECKS == '' ]]; then
+        LINT_STRING="--lint_out=."
+    else
+        LINT_STRING="--lint_out=$LINT_CHECKS:."
+    fi
```

Comment:
There might be a better way to handle this inner if statement, but I'm no bash expert 😄